Plugin~/WebRTCPlugin/WebRTCPlugin.cpp

```diff
@@ -30,111 +32,124 @@ namespace WebRTC
     }
 }
 
+template<class T>
+T** ConvertArray(std::vector<rtc::scoped_refptr<T>> vec, int* length)
+{
+#pragma warning(suppress: 4267)
+    *length = vec.size();
+    const auto buf = CoTaskMemAlloc(sizeof(T*) * vec.size());
+    const auto ret = static_cast<T**>(buf);
+    for (uint32_t i = 0; i < vec.size(); i++)
```

Comment:
Can we  use memory copy here instead of looping ?

Reply:
Like this?

```c++
std::copy(vec.begin(), vec.end(), ret);
```

Reply:
Yes

Reply:
Fixed it
https://github.com/Unity-Technologies/com.unity.webrtc/pull/81/commits/4e8382c7903a27757bec78a2f4d5b22abcbbd454